app/services/idv/actions/verify_document_action.rb

```diff
@@ -0,0 +1,44 @@
+module Idv
+  module Actions
+    class VerifyDocumentAction < Idv::Steps::DocAuthBaseStep
+      def call
+        enqueue_job
+      end
+
+      private
+
+      def form_submit
+        json = form.submit
+        render_json(json, status: form.status)
+        json
```

Comment:
Do you think it would make sense or be much more work to return the same response structure as from the status endpoint? As implemented, and considering support for the existing response structure for image upload controller, there's a lot of differences in the responses the client can receive.

I expect that the client will need to handle the response of `verify_document` and `verify_document_status` basically the same: if the response of either returns a `status` of `in_progress`, wait and try again in a few seconds.

Reply:
I think it seems reasonable to make the API structures match, and luckily we did build that presenter class for the controller

Reply:
> I think it seems reasonable to make the API structures match, and luckily we did build that presenter class for the controller

Yeah, I like this idea of consolidating. I think the presenter can mostly work, though perhaps its naming `ImageUploadResponsePresenter` doesn't align very well to this action, and one or the other of the presenter and action will require a few revisions:

- `status` field doesn't currently exist in presenter response
- `success`: Currently, VerifyDocumentStatusAction treats a failed verification as `success: true` with `status: 'fail'`. `ImageUploadResponsePresenter` and the client application treat verification failure as `success: false`.

Reply:
> I think it seems reasonable to make the API structures match, and luckily we did build that presenter class for the controller

Starting to explore this at #4335 (7a80d376b8155e6da072331c9bfae3ace575fe32). It seems to work okay, though not as clean a fit as I'd have hoped. May still consider renaming `ImageUploadResponsePresenter` to something more descriptive of handling all the responses the client is expecting to receive.

Currently, I haven't needed `status`, between: Using 202 HTTP code for "in progress", and setting `success: false` on failed verification.